pkg/libinit/root_linux.go

```diff
@@ -233,25 +234,81 @@ func CreateRootfs() {
 	}
 }
 
-var excludedMods = map[string]bool{
-	"idpf":     true,
-	"idpf_imc": true,
+// InitModuleLoader wraps the resources we need for early module loading
```

Comment:
one little nit: it's frequently best to define an interface, and the create types like this struct that implement the interface. Why? b/v freebsd has modules, just not modules like this. Also, mocking for testing is easier. You can see this in https://github.com/u-root/u-root/blob/main/pkg/efivarfs/vars_test.go#L16, where I was able to put together things that allowed tests to be easy.